templates/footer.html

```diff
@@ -0,0 +1,14 @@
+<div class="docs-rs-footer">
+    <a href="/about">
+        <span title="About">{{ "info-circle" | fas }}</span>
+        <span class="title">About docs.rs</span>
+    </a>
+    <a href="https://foundation.rust-lang.org/policies/privacy-policy/#docs.rs">
+        <span title="Privacy">{{ "shield-alt" | fas }}</span>
+        <span class="title">Privacy policy</span>
```

Comment:
Why add a separate span instead of using `<a title="Privacy policy>`?
```suggestion
    <a title="Privacy policy" href="https://foundation.rust-lang.org/policies/privacy-policy/#docs.rs">
        {{ "shield-alt" | fas }} Privacy policy
```

Reply:
It's done like this in the navbar so I replicated the same behavior.